pages/secret/[resultId].tsx

```diff
@@ -62,13 +63,17 @@ export async function getServerSideProps(context: {
 }) {
     const { params } = context;
 
-    const res = await fetch(
-        `${process.env.NEXT_PUBLIC_SITE_URL}/api/duckdam/${params.resultId}`
-    );
-    const data = await res.json();
+    // const res = await fetch(
+    //     `${process.env.NEXT_PUBLIC_SITE_URL}/api/duckdam/${params.resultId}`
+    // );
+    // const data = await res.json();
+
+    let result;
+    const url = `${process.env.NEXT_PUBLIC_SITE_URL}/api/duckdam/${params.resultId}`;
+    axios.get(url).then(({ data }) => (result = data));
```

Comment:
요 부분은 `async`, `await`을 활용해서 요런 식으로 바꿔도 될 것 같아요!

```suggestion
    const url = `${process.env.NEXT_PUBLIC_SITE_URL}/api/duckdam/${params.resultId}`;
    const { data } = await axios.get(url);
```

이렇게 하면 아래 요 `props: { data, id: params.resultId }`
`props`를 넘기는 코드를 변경하지 않고도 사용할 수 있을 것 같네용!